logzio-sender/src/main/java/io/logz/sender/LogzioSender.java

```diff
@@ -163,7 +173,7 @@ public void drainQueueAndSend() {
 
     public void send(JsonObject jsonMessage) {
         // Return the json, while separating lines with \n
-        logsBuffer.enqueue((jsonMessage+ "\n").getBytes());
+        logsBuffer.enqueue((jsonMessage+ "\n").getBytes(Charset.forName("UTF-8")));
```

Comment:
Add a test that fails on that (in a separate PR) - copy paste Chinese words from Google, and see that you are receiving the same string. Make sure listener mock works in UTF-8. After your fix, you should see test passes.